concepts/ipc.md

```diff
@@ -53,14 +81,14 @@ There are multiple different ways of implementing ipc on posix systems.
 * D-Bus should offer a high level, easy to use ipc mechanism
```

Comment:
The word "easy" must not be contained in any of our documents. Please think about another formulation :)

Reply:
"simple"